src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -72,5 +72,5 @@ int MinorVersion = 14;
 int BuildNumber = 0;
 // The datetime baseline we choose using this algorithm will affect build number and all nuget packages uploaded
 // Please only change when major or minor version got incremented
-int RevisionNumber = ((int)(DateTime.UtcNow - new DateTime(2021,1,1)).TotalDays)*10+((int)DateTime.UtcNow.Hour)/3;
-#>
\ No newline at end of file
+int RevisionNumber = ((int)(DateTime.UtcNow - new DateTime(2021,1,1)).TotalDays)*10+(int)DateTime.UtcNow.Hour;
```

Comment:
@QilongTang The reference year `DateTime(2021,1,1)` should probably be updated to 2022 the next time there's a build minor bump. I'm not changing it now as it will be confusing if the version is suddenly reset to a smaller number